pyhive/presto.py

```diff
@@ -80,7 +82,10 @@ class Cursor(common.DBAPICursor):
 
     def __init__(self, host, port='8080', username=None, catalog='hive',
                  schema='default', poll_interval=1, source='pyhive', session_props=None,
-                 protocol='http', password=None, requests_session=None, requests_kwargs=None):
+                 protocol='http', password=None, KerberosRemoteServiceName=None,
+                 KerberosPrincipal=None, KerberosConfigPath=None, KerberosKeytabPath=None,
```

Comment:
Is there a reason not to use snake case?

Reply:
Keeping it the same as mentioned in presto docs is the motive.